assets/styles/themes/delivery/adaptive_themes/default/activities/fill-blanks.scss

```diff
@@ -0,0 +1,158 @@
+.ql-container.ql-snow {
```

Comment:
What are you thoughts on nesting / making more specific all these classes with something like `.fib-container` or `janus-fill-blanks`? I'm slightly concerned that they could spill over to unintended elements elsewhere in a lesson...